search/search_param_types.go

```diff
@@ -289,6 +296,24 @@ func (q *Query) UsesPipeline() bool {
 	return q.UsesIncludes() || q.UsesRevIncludes() || q.UsesChainedSearch() || q.UsesReverseChainedSearch()
 }
 
+// SupportsPaging returns true if the query results can be paginated, false if not.
+// In practice, most queries can be paginated, but queries for $everything or _summary cannot.
+func (q *Query) SupportsPaging() bool {
+	options := q.Options()
+
+	// not for $everything. $everything is defined as _id=<id>&_include=*&_revinclude=*
+	if options.IsIncludeAll && options.IsRevincludeAll && strings.Contains(q.Query, "_id") {
```

Comment:
Hmm, good point. My definition of `$everything` never considered an `or` on `_id`. I'll add it!